system/jlib/jthread.cpp

```diff
@@ -1954,6 +1954,7 @@ protected: friend class PipeWriterThread;
             execvp(argv[0],argv);
             _exit(START_FAILURE);    // must be _exit!!     
         }
+        free(argv);
```

Comment:
I don't think so...?
argv is malloc to include space for line + array elements.
The args are memcpy'd in and the argv[<n>] members point to it.